common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -2684,6 +2684,26 @@ def copy_all_asset_metadata(self, source_course_key, dest_course_key, user_id):
                 # update the index entry if appropriate
                 self._update_head(dest_course_key, index_entry, dest_course_key.branch, new_structure['_id'])
 
+    def force_publish_course(self, course_locator, user_id):
+        """
+        Helper method to forcefully publish a course,
+        making the published branch point to the same structure as the draft branch.
+        """
+        print 'Publishing the course {0} ...'.format(course_locator)
```

Comment:
We shouldn't have print statements in this method. If you want, you can have it return some value, `updated`, which is a boolean reflecting whether or not a branch was updated.